designer/client/src/components/ComponentCreate/__snapshots__/ComponentCreateList.test.tsx.snap

```diff
@@ -31,79 +31,103 @@ exports[`ComponentCreateList should match snapshot 1`] = `
           class="govuk-list"
         >
           <li>
-            <a
```

Comment:
isn't this file a test artefact, rather than something that needs committing? I realise you've only modified it not introduced it though :-)

Reply:
Yeah it is, but you're supposed to commit them 😣

I'd prefer if we removed it at some point, but it's generated by one test:

```js
expect(asFragment()).toMatchSnapshot()
```

Whenever you change the source markup, Jest complains that the snapshot needs updating via:

```console
npm test -- -u
```

Reply:
My apologies! So they are.